Shared/Tests.Sync.Shared/Constants.cs

```diff
@@ -0,0 +1,30 @@
+﻿////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+namespace Tests.Sync
+{
+    public static class Constants
```

Comment:
What is the purpose of these constants? Even though they are only in tests, I think they should have some docs because their values embody **some** assumptions. 

For example, are they a user which is expected to exist or **not** to exist?